chart/values.yaml

```diff
@@ -62,6 +62,7 @@ controller:
     image:
       repository: gcr.io/kubebuilder/kube-rbac-proxy
       tag: v0.14.4
+      pullPolicy: Always
```

Comment:
This will introduce a behavioral change, as the default pull policy is `IfNotPresent`.  We can set the default of `IfNotPresent` here, or leave it empty and use default from a helper.

Another idea would be to add a global image pull policy, so all policies could be set from a single value. I think that would probably be our preferred approach, here.

Reply:
ok, I will set pullPolicy in values.yaml to "IfNotPreset"

Reply:
I set it to "IfNotPresent"